eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/processor/BatchPublishMessageProcessor.java

```diff
@@ -103,7 +104,7 @@ public void process(BatchMessage message, EventEmitter<Response> emitter) throws
 
         for (CloudEvent event : cloudEvents) {
             String seqNum = event.getId();
-            String uniqueId = (event.getExtension(ProtocolKey.UNIQUE_ID) == null) ? "" : event.getExtension(ProtocolKey.UNIQUE_ID).toString();
+            String uniqueId = (event.getExtension(ProtocolKey.UNIQUE_ID) == null) ? "" : Objects.requireNonNull(event).getExtension(ProtocolKey.UNIQUE_ID).toString();
```

Comment:
This line is too long and causes the checkstyle to not pass, so you can change the line.

Reply:
changed the line in this [pr](https://github.com/nikam14/incubator-eventmesh/pull/1). 
waiting for your respose to make PR to main repo.